modules/virt-aws-bm.adoc

```diff
@@ -68,3 +68,8 @@ AWS bare-metal and ROSA clusters might have different supported storage solution
 // Changed from "not supported" to "not recommeded" and added "shatred storage" per Ronen Sde-Or 1/31/24 Rosa Virtualization review meeting.
 * Amazon Elastic File System (EFS) and Amazon Elastic Block Store (EBS) are not recommended for use with {VirtProductName} due to performance and functionality limitations. Use shared storage instead.
 --
+
```

Comment:
This addition looks good. 

One thing to consider: this module is reused in the OSD and ROSA docs, but I don't think the addition applies to those distros. HCP has not been implemented at all in OSD, and ROSA with HCP is essentially a different implementation than what's available in OCP. I suggest asking your SMEs if this addition should apply to OSD and ROSA, and if it should be stated differently.

Reply:
@davidvossel @stesrn Please review. Should this only be in virt docs?

Reply:
@davidvossel @stesrn Going to update this to show only for virt. Please open a new JIRA if this needs to be updated to show in other distros.